src/carma-1-tenth/check_message_published.py

```diff
@@ -0,0 +1,27 @@
+# Check that a message exists in the bag
```

Comment:
Is this checking for a ros message or a topic?

Reply:
This wasn't super clear. It checks that there is at least one message on the specified topic. I updated the string to reflect this